docs/section10/10-2.md

```diff
@@ -0,0 +1,41 @@
+**Appendix G Section Reference:** None
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. The zone in the P-RMR includes humidification.
+
+**Function Call:** 
+
+1. match_data_element()
+2. get_zones_hvac_system_humid_p()
+
+**Manual Check:** Check that the applicable baseline HVAC systems were modeled per the rules of G3.1-10 HVAC System baseline column exception which states that if the proposed building humidification system complies with Section 6.5.2.4 then the baseline building design shall use nonadiabatic humidification. For each applicable zone (i.e. zones with humidification in the proposed) check if the humidification system associated with the zone complies with 90.1-2019 Section 6.5.2.4. If it does then check that the baseline system serving the zone was modeled with nonadiabatic humidification. 
+
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Call function to get list of applicable zones in P_RMR: `applicable_zones_hvac_sys_humid_dict_p = get_zones_hvac_system_humid_p(P_RMR)`
+- For each zone in P_RMR: `zone_p in P_RMR...zones:`
+    - Check if the zone has humidification in the P_RMR, if so then applicability flag is true: `if applicable_zones_hvac_sys_humid_dict_p.has_key(zone_p.id): rule_applicability_flag = TRUE`
+    - If rule_applicability_flag = TRUE for zone_p: `if rule_applicability_flag == TRUE:`
+        - Get analogous zone ID in the B_RMR: `zone_b = match_data_element(B_RMR,Zones,zone_p.id)`
+        - For each terminal unit associated with the zone_b in the B_RMR: `for terminal_b in zone_b:`
+            - Get the served_by_heating_ventilation_air_conditioning_systems for each terminal: `heating_ventilation_air_conditioning_systems_b = terminal_b.served_by_heating_ventilation_air_conditioning_systems`
+            - Add to list of heating_ventilation_air_conditioning_systems_list_b as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_b = heating_ventilation_air_conditioning_systems_list_b.append(heating_ventilation_air_conditioning_systems_b)`                
+- Convert the list of heating_ventilation_air_conditioning_systems_list_b associated with the applicable zones to a set and the back to a list to eliminate duplicates after looping through all zones: `heating_ventilation_air_conditioning_systems_list_b = list(set(heating_ventilation_air_conditioning_systems_list_b))`
+- For each hvac_b in heating_ventilation_air_conditioning_systems_list_b: `For in heating_ventilation_air_conditioning_systems_list_b:` 
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, outcome is undetermined, manual check needs to be performed: `UNDETERMINED`
+    
```

Comment:
@claperle and @mkarpman same comment as 10-1. Maybe we can check that baseline and H if the proposed has it and then require manual check to verify the H type.